### PR TITLE
Improve Android docs with missing features or fixes

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

File: src/platforms/android/common/proguard.mdx [hunk 1]
```diff
@@ -127,39 +127,6 @@ sentry {
 
 Now when you build your app, the plugin will upload the ProGuard/R8 mappings, source bundle, and native symbols as you configured them to Sentry.
 
-### Multi-Dex Support
```
Comment (marandaneto) [inline src/platforms/android/common/proguard.mdx:130]: already available via https://docs.sentry.io/platforms/android/configuration/multi-dex/

File: src/includes/getting-started-primer/android.mdx [hunk 2]
```diff
@@ -10,16 +10,28 @@ The SDK builds a crash report that persists to disk and tries to send the report
...
+- Offline caching when a device is offline; we send a report once the App. is restarted
 - [Breadcrumbs automatically](/platforms/android/enriching-events/breadcrumbs/#automatic-breadcrumbs) captured for:
     - Android activity lifecycle events
     - Application lifecycle events (lifecycle of the application process)
     - System events (low battery, low storage space, airplane mode started, shutdown, changes of the configuration, and so forth)
     - App. component callbacks
+    - Android fragment lifecycle events via [Fragment Integration](/platforms/android/configuration/integrations/fragment/)
+    - OkHttp requests via [OkHttp Integration](/platforms/android/configuration/integrations/okhttp/)
+    - Apollo requests via [Apollo Integration](/platforms/android/performance/instrumentation/apollo/)
+    - Timber logs via [Timber Integration](/platforms/android/configuration/integrations/timber/)
```
Comment (imatwawana) [inline src/includes/getting-started-primer/android.mdx:22]: We avoid Latin words/abbreviations like via, etc, e.g., i.e., 

```suggestion
    - Android fragment lifecycle events with [Fragment Integration](/platforms/android/configuration/integrations/fragment/)
    - OkHttp requests with [OkHttp Integration](/platforms/android/configuration/integrations/okhttp/)
    - Apollo requests with [Apollo Integration](/platforms/android/performance/instrumentation/apollo/)
    - Timber logs with [Timber Integration](/platforms/android/configuration/integrations/timber/)
```

File: src/platforms/android/common/enriching-events/context/default-context.mdx [hunk 3]
```diff
@@ -27,6 +27,7 @@ Package info
 - Bundle ID
 - Version
 - Build
+- Whether the App. is Sideloaded
```
Comment (imatwawana) [inline src/platforms/android/common/enriching-events/context/default-context.mdx:30]:
```suggestion
- Whether the application is sideloaded
```

File: src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 4]
```diff
@@ -124,7 +124,7 @@ Learn more in our [Fragment documentation](/platforms/android/configuration/inte
 
 ### App Start Instrumentation
 
-The App Start Instrumentation provides insight into how long your application takes to launch. It adds a span from the application launch to the first auto-generated UI transaction.
+The App Start Instrumentation provides insight into how long your application takes to launch. It adds a span from the application launch to the first auto-generated UI transaction. The SDK sets the Span operation to `app.start.cold` or `app.start.warm`.
```
Comment (imatwawana) [inline src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx:127]:
```suggestion
The App Start Instrumentation provides insight into how long your application takes to launch. It adds a span from the application launch to the first auto-generated UI transaction. Also, the SDK sets the Span operation to `app.start.cold` or `app.start.warm`.
```

**Review by imatwawana — APPROVED** [review]
Some minor style and wording changes

**Comment by marandaneto** [comment]
thanks @imatwawana

File: src/platforms/android/common/proguard.mdx (hunk 1 above)
Comment (philipphofmann) [reply src/platforms/android/common/proguard.mdx:130]: Should we then maybe add a link here instead?

File: src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 5]
```diff
@@ -124,7 +124,7 @@ Learn more in our [Fragment documentation](/platforms/android/configuration/inte
 
 ### App Start Instrumentation
 
-The App Start Instrumentation provides insight into how long your application takes to launch. It adds a span from the application launch to the first auto-generated UI transaction.
+The App Start Instrumentation provides insight into how long your application takes to launch. It adds a span from the application launch to the first auto-generated UI transaction. Also, the SDK sets the Span operation to `app.start.cold` or `app.start.warm`.
```
Comment (philipphofmann) [inline src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx:127]: `m`: I'm curious on why you put that sentence here? The docs mention cold and warm start directly below. This seems to be a bit redundant information to me.

File: src/includes/getting-started-primer/android.mdx [hunk 6]
```diff
@@ -10,16 +10,28 @@ The SDK builds a crash report that persists to disk and tries to send the report
...
 - [User Feedback](/platforms/android/enriching-events/user-feedback/) provides the abiity to collect user information when an event occurs.
-- [Performance Monitoring](/product/performance/) creates transactions
+- [Performance Monitoring](/product/performance/) creates transactions for:
+    - Android activity transactions
+    - Android fragment spans with [Fragment Integration](/platforms/android/configuration/integrations/fragment/)
+    - Cold and warm app start spans and measurements with [Automatic Instrumentation](/platforms/android/performance/instrumentation/automatic-instrumentation/#app-start-instrumentation)
+    - Slow and frozen frames measurements with [Automatic Instrumentation](/platforms/android/performance/instrumentation/automatic-instrumentation/#slow-and-frozen-frames)
+    - OkHttp request spans with [OkHttp Integration](/platforms/android/configuration/integrations/okhttp/)
+    - SQLite and Room query spans with [Sentry Android Gradle Plugin](/platforms/android/performance/instrumentation/automatic-instrumentation/#sqlite-and-room-instrumentation)
```
Comment (philipphofmann) [inline src/includes/getting-started-primer/android.mdx:32]: `m`: Maybe worth mentioning here that this is still in beta?

**Review by philipphofmann — COMMENTED** [review]
Looking good. I added a few minor comments.

File: src/platforms/android/common/proguard.mdx (hunk 1 above)
Comment (marandaneto) [reply src/platforms/android/common/proguard.mdx:130]: no, multi-dex isn't proguard related

File: src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx (hunk 5 above)
Comment (marandaneto) [reply src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx:127]: every auto instrumentation should mention the operation names, so people can look at it and filter, if here is the right place, maybe you are right, we could mention somewhere else

File: src/includes/getting-started-primer/android.mdx (hunk 6 above)
Comment (marandaneto) [reply src/includes/getting-started-primer/android.mdx:32]: its a link to the page, when somebody clicks on it, they will read the beta note

File: src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx (hunk 5 above)
Comment (marandaneto) [reply src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx]: moved to a new line, thanks

**Review by romtsn — APPROVED** [review]

**Pull request merged**